### PR TITLE
Disable flaky test retry in flexCI

### LINKED ISSUE #8129: Remove automatic retry for flaky tests
As the flaky test issues are mostly resolved (#6903), we decided to try disabling automatic retry in flexCI.

### PR DESCRIPTION
**Pull request by kmaehashi**
Close #8129

### CODE REVIEW

File: .pfnci/hint.pbtxt [hunk 1]
```diff
@@ -3,22 +3,20 @@
...
 # https://github.com/chainer/xpytest/blob/master/proto/test_case.proto
 
-# Too flaky tests.  Retry up to 4 times.
-rules { name: "chainerx_tests/unit_tests/routines_tests/test_arithmetic.py" xdist: 8 deadline: 240 retry: 4 }
-rules { name: "chainerx_tests/unit_tests/routines_tests/test_math.py" xdist: 8 deadline: 240 retry: 4 }
-rules { name: "chainer_tests/functions_tests/activation_tests/test_elu.py" xdist: 2 retry: 4 }
-
 # Slow tests take 60+ seconds.
 rules { name: "chainer_tests/functions_tests/activation_tests/test_crelu.py" xdist: 4 deadline: 240 }
+rules { name: "chainer_tests/functions_tests/activation_tests/test_elu.py" xdist: 2 }
```
Comment (niboshi) [inline .pfnci/hint.pbtxt]: Shouldn't this entry be removed?

Comment (toslunar) [reply .pfnci/hint.pbtxt]: > Shouldn't this entry be removed?

Is the test fast enough?

Comment (niboshi) [reply .pfnci/hint.pbtxt]: I don't know, but this entry doesn't have `deadline` anyway.

Comment (toslunar) [reply .pfnci/hint.pbtxt]: It seems "Slow tests take 30+ seconds" use `xdist: 2`.

**Comment by niboshi** [comment]
Jenkins, test this please

File: .pfnci/hint.pbtxt (hunk 1 above)
Comment (kmaehashi) [reply .pfnci/hint.pbtxt]: I measured and the test takes 30+ seconds.

**Pull request merged**